messenger.rst

```diff
@@ -924,12 +924,29 @@ stream_max_entries  The maximum number of entries which    ``0`` (which means "n
                     the stream will be trimmed to. Set
                     it to a large enough number to
                     avoid losing pending messages
+redeliver_timeout   Timeout before retrying a pending       3600
+                    message which is owned by an
+                    abandoned consumer (if a worker died
+                    for some reason, this will occur,
+                    eventually you should retry the
+                    message) - in seconds.
+claim_interval      Interval on which pending/abandoned     1000
+                    messages should be checked for to
+                    claim - in milliseconds
 ==================  =====================================  =========================
 
 .. versionadded:: 4.4
 
     The ``stream_max_entries`` option was introduced in Symfony 4.4.
 
+.. caution::
+
+    There should never be two or more `messenger:consume` commands running with the same
+    config (stream, group and consumer name) to avoid having a message handled more than once.
+    Using the ``HOSTNAME`` as the consumer might often be a good idea. In case you are using
+    Kubernetes to orchestrate your containers, consider using a ``StatefulSet``.
```

Comment:
Think this is what @toflar wanted to document in #11869. ping @Toflar anything you want to add here?

Reply:
Well, with this PR it's not really relevant anymore. You can use a regular `Deployment` which changes hostnames after restarts but the PR is here to fix exactly that. The `StatefulSet` is just so that you always get the same hostnames back.

Reply:
Yeah thats correct still think its good to have the hostname which doesn't change as it avoids spaming redis with new consumers as they recommend to reuse consumer names.

Reply:
Then that should probably be mentioned here but other than that, there's nothing to add regarding k8s :)